tests/lvm_utils.go

```diff
@@ -124,11 +149,34 @@ func extendVg(name string, device string) {
 // Does vhremove on specified vg with force flag,
 // lv will be forcedeleted if vg is not empty.
 func removeVg(name string) {
-	fmt.Printf("Removing vg\n")
+	ginkgo.By("Removing vg")
+	retries := 5
+	current_retry := 0
+	force := false
+	for {
+		if current_retry < retries {
+			vg_empty := vgEmpty(name)
+			if vg_empty {
+				fmt.Printf("No lv in vg before vg remove\n")
+				break
+			} else {
+				fmt.Printf("lv in vg during retry %d\n", current_retry)
+			}
+		} else {
+			fmt.Printf("vg still not empty after 5 seconds, moving on with force delete\n")
+			force = true
+			break
+		}
+		current_retry += 1
+		time.Sleep(2 * time.Second)
+	}
+
 	args_vg := []string{
 		"vgremove",
 		name,
-		"-f",
+	}
+	if force {
+		args_vg = append(args_vg, "-y")
```

Comment:
Do you intend to append `-f` here and not `-y` ?

Reply:
-y is yes for all. If lv is in vg while vgremove it asks whether to remove lv. So used -y.

Reply:
ill use -f